habitTrackerApp.py

```diff
@@ -24,18 +24,17 @@
 from kivy.core.window import Window
 from kivy.uix.checkbox import CheckBox
 
-Window.left = -1000
+# Window.left = -1000
```

Comment:
Always add a comment to why a code has been commented. 
We call this dead code and it can be be pretty noisy as someone not familiar with the codebase might be wondering why it's there but not used